app/components/events.js

```diff
@@ -0,0 +1,15 @@
+import Component from '@glimmer/component';
+import { inject as service } from '@ember/service';
+import { EVENTS_CATEGORIES } from '../constants/events-data';
+
+export default class EventsComponent extends Component {
+  @service router;
+  EVENTS_CATEGORIES = EVENTS_CATEGORIES;
+
+  get isDev() {
+    if (this.router.currentRoute) {
+      return this.router.currentRoute.queryParams.dev;
+    }
+    return false;
+  }
+}
```

Comment:
Why you don't used service file feature flag??

Reply:
Didn't notice the service, saw another component using this approach so had gone with this. Changed to service file now.